src/styles.css

```diff
@@ -1,4 +1,3 @@
-/* font-family: 'Lora', serif; */
 /* ~~~~~~~~~~ BASIC ELEMENTS ~~~~~~~~~~~~ */
```

Comment:
Really appreciate the headers to each section delineating what's going on in there.